database/factories/RoomFactory.php

```diff
@@ -25,7 +25,7 @@ public function definition()
             'name' => $this->faker->word,
             'number' => $this->faker->word,
             'floor' => $this->faker->RandomDigit,
-            'building' => $this->faker->word,
+            'building' => $this->faker->randomElement(Room::BUILDING_NAMES),
```

Comment:
pretty sure you can remove the `$this->faker` since the next part will just get a random name anyway

Reply:
@EvanDime what do you mean? I'm not sure I understand

Reply:
this was my mistake. I thought randomElement was a generic laravel function, but it's a faker method.